src/lib/ng-package/package.ts

```diff
@@ -88,3 +88,5 @@ export class NgPackage {
     return [this.primary, ...this.secondaries].find(entryPoint => entryPoint.moduleId === moduleId);
   }
 }
+
+export type AssetEntry = { glob: string; input: string; ignore?: string[]; output: string };
```

Comment:
Can you change this to an interface please since in this case this describes an object which interfaces are intended for.

Reply:
Done